components/core/blocks/Auth/SignUp.vue

```diff
@@ -22,9 +19,9 @@ export default {
   },
   components: {
     Modal,
-    Login,
-    Register,
-    ForgotPass
+    Login: () => import('theme/components/core/blocks/Auth/Login'),
```

Comment:
please add names chunks :pray: `/* webpackChunkName: "modal-auth-login" */ `
It's useful for two things. First it's easier to know what we load in browser. Second we can use initialResources. For example:
```

    {
      "filters": ["modal-auth-login"],
      "type": "script",
      "onload": false,
      "rel": "prefetch"
    }
```
and we can get rid of it